python/paddle/distribution/normal.py

```diff
@@ -95,7 +95,7 @@ class Normal(distribution.Distribution):
     """
 
     def __init__(self, loc, scale, name=None):
-        if not in_dynamic_mode():
+        if not in_dynamic_or_pir_mode():
```

Comment:
```suggestion
        if not in_dynamic_mode():
```
这里不需要更改。因为 pir 也属于静态图模式，可以复用静态图的 check_type 的分支。但是这个判断分支下面的 check_type 的类型需要添加类型 `paddle.base.libpaddle.pir.Value`

Reply:
好滴~